controller/src/main/java/org/jboss/as/controller/SimpleResourceDefinition.java

```diff
@@ -64,10 +64,11 @@
      * @param pathElement         the path. Can be {@code null}.
      * @param descriptionProvider the description provider. Cannot be {@code null}
      * @throws IllegalArgumentException if {@code descriptionProvider} is {@code null}.
-     * @deprecated
+     * @deprecated Use {@link #SimpleResourceDefinition(Parameters)}
      */
     @Deprecated
     public SimpleResourceDefinition(final PathElement pathElement, final DescriptionProvider descriptionProvider) {
+        //Can be removed when we get to 3.0.0
```

Comment:
That will require rewriting a ton of existing subclasses, and we don't have time for that.